src/pcapi/notifications/push/__init__.py

```diff
@@ -9,6 +9,11 @@ def update_user_attributes(user_id: int, attribute_values: dict) -> None:
     backend().update_user_attributes(user_id, attribute_values)
 
 
+def update_user_attributes_with_legacy_internal_task(user_id: int, attribute_values: dict) -> None:
+    backend = import_string(settings.PUSH_NOTIFICATION_BACKEND)
+    backend().update_user_attributes_with_legacy_internal_task(user_id, attribute_values)
```

Comment:
```suggestion
def update_user_attributes_with_legacy_internal_task(user_id: int, attribute_values: dict) -> None:
    backend = import_string(settings.PUSH_NOTIFICATION_BACKEND)
    backend().update_user_attributes(user_id, attribute_values)
```

les méthodes update_user_attributes_with_legacy_internal_task des 2 backends servent un peu à rien non ?
Quitte à avoir ce wrapper autant qu'il puisse choisir la bonne méthode sans rajouter une couche en dessous ?